src/cli/server/mod.rs

```diff
@@ -16,6 +17,15 @@ fn log_launch(sc: &ServerConfig) {
     );
 
     let url = sc.graphiql_url();
-    let url = format!("https://tailcall.run/playground/?u={}/graphql", url);
+    let utm_source = if VERSION.eq("0.1.0-dev") {
+        "tailcall-debug"
+    } else {
+        "tailcall-release"
```

Comment:
Can create a `playground.rs` file that builds the URL. Move all the query params to `const` in that file.